airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -385,18 +385,25 @@ def attempt_task_runs(self):
                     )
                     self.pending_tasks.append(ecs_task)
                 else:
-                    self.send_message_to_task_logs(
-                        logging.ERROR,
-                        "ECS task %s has failed a maximum of %s times. Marking as failed. Reasons: %s",
-                        task_key,
-                        attempt_number,
-                        ", ".join(failure_reasons),
-                        ti=task_key,
+                    reasons_str = ", ".join(failure_reasons)
+                    self.log_task_event(
+                        record=Log(
+                            event="ecs executor queue error",
```

Comment:
```suggestion
                            event="ecs executor task error",
```

Reply:
isn't it an error specifically with regard to executor trying to queue this task?  separately, the reason i didn't put "task" here and elsewhere is because it's already clear from the log record that it is specific to a task so i think it does not need to be stated and we can just assume it's task and be specific about the type of task error.  in the context where this is presented in the UI, we are showing all events related to the task.

Reply:
`task` is definitely overused xD Here I am referring to ECS task, which is the name used in ECS to refer to a job. But here similarly to my previous comment, we are dequeing jobs from a queue and try to execute them. This execution fail but it is not a queue failure to me

Reply:
going with `ecs task submit failure`